Modix/Modules/RandomNumberModule.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace Modix.Modules
```

Comment:
You could group this inside a `Random` module have have two separate methods within the same module